Library/Homebrew/formula_installer.rb

```diff
@@ -214,6 +215,10 @@ def install
       opoo "#{formula.full_name}: #{old_flag} was deprecated; using #{new_flag} instead!"
     end
 
+    invalid_opt_names.each do |option|
+      opoo "#{formula.full_name}: #{option} is invalid for this formula and will be ignored!"
```

Comment:
`opoo "#{formula.full_name}: this formula has no #{option} option so it will be ignored!"`